code/modules/mob/mob_helpers.dm

```diff
@@ -685,3 +685,25 @@ var/global/image/backplane
 		return
 
 	item.screen_loc = screen_place
+
+/*
+ * Generalized proc for moving a player to another mob.
+ * 
+ * Returns FALSE if it is unable to transfer the player.
+ * allow_overwrite determines if it will fail when targeting a mob with a ckey.
+ */
+/mob/proc/transfer_player(var/mob/target = null, var/allow_overwrite = FALSE)
+	if(!target)
+		return FALSE
+
+	if(target.ckey && !allow_overwrite)
+		return FALSE

```

Comment:
So we can't do it even if they logged out?

Reply:
I don't know if there's a way of _determining_ that.

Reply:
`target.client`, I think?

Reply:
`target.client` is null and `target.ckey` is not if they have logged out.